exchanges/binance/binance_wrapper.go

```diff
@@ -401,7 +401,12 @@ func (b *Binance) FetchOrderbook(p currency.Pair, assetType asset.Item) (*orderb
 
 // UpdateOrderbook updates and returns the orderbook for a currency pair
 func (b *Binance) UpdateOrderbook(p currency.Pair, assetType asset.Item) (*orderbook.Base, error) {
-	book := &orderbook.Base{ExchangeName: b.Name, Pair: p, AssetType: assetType}
+	book := &orderbook.Base{
+		ExchangeName:       b.Name,
+		Pair:               p,
+		AssetType:          assetType,
+		VerificationBypass: b.OrderbookVerificationBypass,
+	}
```

Comment:
This is good for REST, however, for the websocket, the `b.OrderbookVerificationBypass` isn't being set in `SeedLocalCacheWithBook`/`SeedLocalCache`.

It's also happening for ZB I've noticed, so please double check websocket implementations. 


I'm just logging every time validation is run for an exchange, so if I see it when I set bypass to true, I know something is up.

Reply:
Will check it out!